doc/classes/Shader.xml

```diff
@@ -4,10 +4,11 @@
 		A custom shader program.
 	</brief_description>
 	<description>
-		This class allows you to define a custom shader program that can be used for various materials to render objects.
+		This class allows you to define a custom shader program that can be used by a [ShaderMaterial]. Shaders allow you to write your own custom behaviour for rendering objects or updating particle information. For a detailed explanation and usage, please see the tutorials linked below.
```

Comment:
Sorry my dear Canadian, your fellow English-speakers on the other side of the International Border sadly won the infowards, and our Us got ripped out of all our OURs... ;) ("behaviour")

Reply:
Whoops! I let my accent slip. :P